### ISSUE TITLE
How add another chain?

### ISSUE THREAD

**Issue by info170** [KelvinThai/PancakeSwapFork#2]
Hi
Thanks for video, I successfully cloned Pancakeswap and ran it.
Also I was need add Arbitrum chain because my own token was created there.
I added it to swap page but couldn't add liquidity.
Can you show please how correctly add new chain to Pancake frontend?

**Comment by KelvinThai** [comment]
it will be more tricky, you need manually deploy your own wrapped tokens, factories, router on the chain before working on the frontend.

**Comment by info170** [comment]
Yeah, I already did it - I have router address, factory address, my new token address, wrapped eth address (for liquidity pair, I guess)
But don't know where correctly insert these settings in frontend.
